buildSrc/src/main/groovy/com/uber/okbuck/experimental/bazel/AndroidLibraryRule.groovy

```diff
@@ -0,0 +1,66 @@
+package com.uber.okbuck.experimental.bazel
+
+import com.uber.okbuck.core.model.AndroidTarget
+import com.uber.okbuck.rule.BuckRule
+
+final class AndroidLibraryRule extends BuckRule {
+    private static final String RULE_TYPE = "android_library"
+
+    private final Set<String> srcSet
+    private final String manifest
+    private final Set<String> providedDeps
+    private final String packageName
+    private final Set<AndroidTarget.ResBundle> resources
+
+    AndroidLibraryRule(
```

Comment:
does bazel not support annotation processors currently? or is it done implicitly somehow?